apps/client/src/services/api.ts

```diff
@@ -0,0 +1,7 @@
+const API_BASE_URL =
+  process.env.NEXT_PUBLIC_API_BASE_URL || "http://localhost:3001";
+
+export const getApiUrl = (path: string, search?: string | null) => {
+  const url = `${API_BASE_URL}${path}`;
+  return search ? `${url}?search=${encodeURIComponent(search)}` : url;
+};
```

Comment:
This probably could have moved to `@/lib/utils` but I felt like keeping this closer to the resource/entity services was better than moving them elsewhere.